src/sentry/static/sentry/app/views/organizationIncidents/list/index.jsx

```diff
@@ -95,9 +94,9 @@ class OrganizationIncidentsList extends AsyncComponent {
             <TableLayout>
               <div>{t('Incident')}</div>
               <div>{t('Status')}</div>
-              <div>{t('Started')}</div>
-              <div>{t('Users affected')}</div>
-              <div>{t('Total events')}</div>
+              <div>{t('Duration')}</div>
```

Comment:
We have the absolute date there, will "duration" be confusing?

Reply:
Ah, looks like the column is now "Started," so i'll update this to match.